modules/microshift-fips-rpm-system.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * microshift_install/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-fips-rpm-system_{context}"]
+= Enabling FIPS compliance with RPM-based installations
+
+Using FIPS with {product-title-first} requires enabling the cryptographic module self-checks in your {op-system-base-full} installation. After the host operating system has been set up to start up with the FIPS modules, {microshift-short} containers are automatically enabled to run in FIPS mode.
+//Q: the RHEL docs say podman does this; true for MicroShift?
+
+* When {op-system-base} is booted in FIPS mode, {microshift} core components use the RHEL cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64 architectures.
+
+* You must enable FIPS mode when you install {op-system-base} {op-system-version-major} on the machines that you plan to use as worker machines.
+
+[IMPORTANT]
+====
+Because FIPS must be enabled before the operating system that your cluster uses boots for the first time, you cannot enable FIPS after you deploy a cluster.
+====
+
+* {microshift-short} uses a FIPS-compatible golang compiler.
+
+* FIPS is supported in the CRI-O container runtime.
+//OCP has a table of attributes and limitations that didn't make a whole lot of sense; do we need to add any limitation or caveat to cri-o container runtime support?
+
+== Limitations
+
+* TLS implementation FIPS support is not complete.
+
+* The FIPS implementation does not offer a single function that both computes hash functions and validates the keys that are based on that hash. This limitation will continue to be evaluated and improved in future {microshift-short} releases.
+//from the OCP docs; are these true for microshift?
+
+== Installing {op-system-base} in FIPS mode
+
+* To install {op-system-base} with FIPS, follow the guidance in the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode] of the {op-system-base} documentation.
+
+* See the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening#federal-information-processing-standards-140-and-fips-mode_assembly_installing-the-system-in-fips-mode[Federal Information Processing Standards 140 and FIPS mode] section of the {op-system-base} for details.
```

Comment:
Unsure about these just being two bullets in this section (without being part of something like a Procedure or Addtl Resources). Could this alternatively be something like:
```suggestion
To install {op-system-base} with FIPS, follow the guidance in the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode] of the {op-system-base} documentation.

[role="_additional-resources"]
.Additional resources
* https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening#federal-information-processing-standards-140-and-fips-mode_assembly_installing-the-system-in-fips-mode[Federal Information Processing Standards 140 and FIPS mode] section of the {op-system-base} documentation
```
That is, make the first bullet a regular sentence, and turn the 2nd "more details" link into an Addtl Resources section.

Even further (tho it's starting to feel silly/pedantic), maybe this should actually be presented as a Procedure, even if it's only 1 step and is basically just linking off:
```suggestion
.Procedure

* To install {op-system-base} with FIPS, follow the guidance in the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode] of the {op-system-base} documentation.

[role="_additional-resources"]
.Additional resources
* https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening#federal-information-processing-standards-140-and-fips-mode_assembly_installing-the-system-in-fips-mode[Federal Information Processing Standards 140 and FIPS mode] section of the {op-system-base} documentation
```
If it does become a proper Procedure, consider also that procedures should be broken out into separate modules.